packages/dds/merge-tree/src/test/client.obliterateFarm.spec.ts

```diff
@@ -29,16 +29,22 @@ interface IObliterateFarmConfig extends IMergeTreeOperationRunnerConfig {
 }
 
 const allOperations: TestOperation[] = [
-	removeWithField,
+	// The test model is not currently stable enough to support removeWithField: logic to look for fields
+	// relies on text inside fields being numeric and text outside of fields alphabetic, but if a removal
+	// operation takes away a whole field, a concurrent field replace (obliterate + insert new field content)
+	// can result in numeric characters outside of fields.
```

Comment:
Not blocking but this seems like a real issue for customers. So they might have to do something like obliterate whole fields when removing ranges that contain them in order to prevent those insertions from showing up.